app/scienceapi/events/models.py

```diff
@@ -27,6 +27,18 @@ class Event(models.Model):
     Science site
     """
     name = models.CharField(max_length=300)
+    category = models.CharField(
+        max_length=50,
+        default='Project Calls',
+        choices=(
```

Comment:
Also, more feedback, the categories are hardcoded here, which limits some scaling, what if we add more?..are these broad enough? I would say that the following would be good starter categories for our events: 

* Project Call
* Study Group Call 
* Community Call
* Workshop
* Sprint
* Mozfest
* Conference
* Meetup
* Convening

Perhaps it seems like a lot, but all of these cover our event hosting needs in broad, and would allow us to add code sprints, community meetups and other convenings later, so the scale seems right. All of them are singular rather than plural, since as a category they will be tagging individual events.